src/Files.App/Actions/Show/ToggleSidebarAction.cs

```diff
@@ -0,0 +1,41 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Commands;
+using Files.App.Extensions;
+using Files.App.ViewModels;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal class ToggleSidebarAction : ObservableObject, IToggleAction
+	{
+		private readonly SidebarViewModel viewModel = Ioc.Default.GetRequiredService<SidebarViewModel>();
+
+		public string Label { get; } = "TogglePreviewPane".GetLocalizedResource();
+
+		public string Description => "TODO: Need to be described.";
+
+		public HotKey HotKey { get; } = new(VirtualKey.S, VirtualKeyModifiers.Control | VirtualKeyModifiers.Menu);
```

Comment:
Meanwhile I had no idea this shortcut existed! Can you use something a little simpler and add it to the documentation?

Reply:
VS Code uses `ctrl` + `b`